src/main/java/team/project/dto/user/UserResponseDto.java

```diff
@@ -0,0 +1,10 @@
+package team.project.dto.user;
+
+import lombok.Data;
+
+@Data
```

Comment:
Don't use this @Data annotation (our mentors always banned it), read it https://medium.com/@miguelangelperezdiaz444/the-hidden-dangers-of-lombok-annotations-in-your-java-code-what-you-need-to-know-8acdce2d6b89
It'll be better to use recorder here instead of class